src/medication-request/medication-request.resource.tsx

```diff
@@ -1,79 +1,26 @@
 import useSWR from "swr";
-import {
-  fhirBaseUrl,
-  openmrsFetch,
-  openmrsObservableFetch,
-} from "@openmrs/esm-framework";
+import { fhirBaseUrl, openmrsFetch } from "@openmrs/esm-framework";
 import {
   AllergyIntoleranceResponse,
+  EncounterWithMedicationRequests,
+  EncountersWithMedicationRequestsResponse,
   MedicationRequest,
   MedicationRequestResponse,
+  Prescription,
 } from "../types";
 
```

Comment:
I extracted all the interfaces below in the "types" file and also renamed them to names that I thought were clearer... but feel free to weigh in if you disagree with the renaming.